CMakeLists.txt

```diff
@@ -4,7 +4,11 @@ project(RecastNavigation)
 
 # lib versions
 SET(SOVERSION 1)
-SET(VERSION 1.0.0)
```

Comment:
Can be simplified a bit:

```cmake
set(LIB_VERSION 1.5.1)
string(REPLACE LIB_VERSION "." "," LIB_VERSION_NUM)
set(LIB_VERSION_NUM, "${LIB_VERSION_NUM},0")
```

Reply:
Thanks, yes it needn't be so verbose. It needn't be exactly like OpenAL-Soft ;)